v22.1/backup-and-restore-overview.md

```diff
@@ -49,6 +49,10 @@ The following table outlines SQL statements you can use to create, configure, pa
 [`RESUME SCHEDULES`](resume-schedules.html) | Resume paused backup schedules.
 [`DROP SCHEDULES`](drop-schedules.html) | Drop backup schedules.
 
+## {{ site.data.products.db }} backup types
+
+{% include cockroachcloud/ccloud/backup-types.md %} 
```

Comment:
At the top of this page, there is a breakout of the major sections 

This page provides an overview of the backup and restore features available in CockroachDB:
[Backup and restore types](https://deploy-preview-15524--cockroachdb-docs.netlify.app/docs/stable/backup-and-restore-overview.html#backup-and-restore-types) and their product availability
[SQL statements](https://deploy-preview-15524--cockroachdb-docs.netlify.app/docs/stable/backup-and-restore-overview.html#backup-and-restore-sql-statements) for working with backups and restores
[Storage](https://deploy-preview-15524--cockroachdb-docs.netlify.app/docs/stable/backup-and-restore-overview.html#backup-storage) for backups

Should we add "Types of backups supported in CockraochDB Cloud" to this list?